roles/ovirt-ova-pack/files/pack_ova.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+
+import os
+import io
+import tarfile
+import sys
+import time
+
+BLOCKSIZE = 512
+NUL = "\0"
+buf = bytearray(4096)
+
+def createTarInfo(name, size):
```

Comment:
Can you please use snake case, instead of camel case?

Reply:
done